### ISSUE TITLE
Appstore homebrew is not updating on both website and switch

### ISSUE THREAD

**Issue by ghost** [vgmoose/hbas#75]
**Describe the bug**
On both the website and on my Switch the list of homebrew isn't being updated. For example hekate is still at 5.5.8. The latest updates are at July 15th (7/15).

**Platform:**
Switch, PC

**To Reproduce**
Navigate to the website of the appstore, or launch the appstore NRO.
See that there are many out-of-date homebrew, and nothing has been updated since July 15, 
![image](https://user-images.githubusercontent.com/77812141/133367510-62405571-0de4-49b2-987f-5178c8e3a2d8.png)
![image](https://user-images.githubusercontent.com/77812141/133367550-f0a64fe3-9292-4a00-83bc-52714088346f.png)

Thank you for your time!

**Comment by mtoensing** [comment]
Same here, Retroarch is very old!

**Comment by Aimela** [comment]
Yeah, I don't get why it's two months old at this point. This is definitely a problem for an appstore like this.

**Comment by TrainDoctor** [comment]
Yeah I'm encountering this issue as well. I think the app's lack of maintenance updates is a bad sign. Hopefully we can see some info soon. 🤞🏻

**Comment by ghost** [comment]
UPDATE: The site has been updated! I can access downloads now!

**Comment by compucat** [comment]
Thanks for your patience, all - apologies for the late reply. Updates to HBAS's repository are processed by hand - this wasn't a CI failure, this was just a case of life getting in the way for a while and us being terrible communicators. We've gone ahead and pushed a new batch of updates through, and we've brought another person on to help keep the pipeline flowing; eventually, we will get around to automating the majority of these updates with Spinarak (the libget autodeployment system I wrote last year).

Closing.
